tcp/pcb.ml

```diff
@@ -465,6 +468,10 @@ struct
       Lwt.return_unit
     | None ->
       Tx.send_rst t id ~sequence ~ack_number ~syn ~fin
+    | exception ex ->
+      PCB_Log.err (fun f -> f "uncaught exception looking up handler for TCP port %d:@ %s"
+                      id.WIRE.local_port (Printexc.to_string ex));
+      Lwt.return_unit
```

Comment:
afaics this case can never happen -- the passed in `listeners` is from [direct stack](https://github.com/mirage/mirage-tcpip/blob/a54f6385a433407a7b29391b148967af66d4f22a/lib/tcpip_stack_direct.ml#L142-L144) and a `try` around `Hashtbl.find`.